Dockerfile

```diff
@@ -0,0 +1,25 @@
+ARG ROS_DISTRO=galactic
+
+FROM ros:$ROS_DISTRO-ros-base
```

Comment:
The image could be based on `ros:$ROS_DISTRO-ros-core` but that would require installing `rosdep` anyway, so it is arguable if it is needed.

Reply:
:heavy_check_mark: